test/cases/ostree-simplified-installer.sh

```diff
@@ -759,6 +759,123 @@ EOF
 sudo ansible-playbook -v -i "${TEMPDIR}"/inventory -e image_type=redhat -e ostree_commit="${INSTALL_HASH}" -e fdo_credential="true" /usr/share/tests/osbuild-composer/ansible/check_ostree.yaml || RESULTS=0
 check_result
 
+##################################################################
+##
+## Build edge-simplified-installer without FDO 
+##
+##################################################################
+
+tee "$BLUEPRINT_FILE" > /dev/null << EOF
+name = "simplified_iso_without_fdo"
+description = "A rhel-edge simplified-installer image without FDO”
```

Comment:
@sarmahaj I think the last double quotes aren't correct - you should just use `"`

Reply:
somehow I missed this quote earlier, thanks for catching this